test/test_jit_nvfuser_graph.py

```diff
@@ -0,0 +1,43 @@
+import torch
```

Comment:
Note, python test should go into https://github.com/pytorch/pytorch/blob/master/test/test_jit_cuda_fuser.py

Reply:
Jie, move to https://github.com/csarofeen/pytorch/blob/devel/test/test_jit_cuda_fuser.py?